src/site/content/en/learn/css/box-model/index.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: course
+title: Box Model
+authors:
+  - andybell
+tags:
+  - css
+---
+
+Say you have this bit of HTML:
+
+```html
+<p>I am a paragraph of text that has a few words in it.</p>
+```
+
+Then say you wrote this CSS for it:
+
+```css
+p {
+  width: 100px;
+  height: 50px;
+  padding: 20px;
+  border: 1px solid;
+}
+```
+
+The content would break out of your element and it would be 140px
+wide, rather than 100px. Why is that? The box model is a core
+foundation of CSS and understanding how it works, how it is affected
+by other aspects of CSS and importantly, how you can control it will
+help you to write more predictable CSS.
+
+<iframe height="500" style="width: 100%;" scrolling="no" title="Learn CSS - Box model intro" src="https://codepen.io/piccalilli/embed/0c03f5dc8a83ccaa2f48fcc115e1ec54?height=500&theme-id=38982&default-tab=result" frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/piccalilli/pen/0c03f5dc8a83ccaa2f48fcc115e1ec54'>Learn CSS - Box model intro</a> by Piccalilli
+  (<a href='https://codepen.io/piccalilli'>@piccalilli</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+---
```

Comment:
So, @robdodson, this whole block here is a specific intro and in content meetings, we've been discussing giving them a specific design treatment. Is it worth thinking about some sort of `{% Intro %}` paired shortcode?

Reply:
sure I'm open to it. any thoughts on what y'all would want the design to look like?